build.gradle

```diff
@@ -21,14 +21,30 @@ buildscript {
 }
 
 repositories {
-	maven {
-		url "https://oss.sonatype.org/content/repositories/snapshots"
-	}
+    mavenLocal()
+    mavenCentral()
```

Comment:
No need anymore, it will take jars from bintray.

Reply:
ok